### ISSUE TITLE
Can we stop using PR diffs for discussions?

### ISSUE THREAD

**Issue by kg** [WebAssembly/design#282]
The comments on diffs get really hard to follow since they go away when the PR changes, and I keep seeing super hosed megaquotes with broken formatting show up in my inbox in an unreadable format. When I try to find them in the PR I can't find them.

If we could stick to doing most of our threaded discussion in the issue section of the PR, or in issues, that problem will go away.

**Comment by creationix** [comment]
I was thinking about this today as well.  I think inline comments in the diff are best for short-lived comments on that code.  Like for example, pointing out a typo.  We actually want our comment to go away once the typo is fixed.

**Comment by qwertie** [comment]
Ahh, I kept wondering why all those messages in my email weren't showing up on the web site!

**Comment by sunfishcode** [comment]
It's a good suggestion. We should try to do this more.

I'm closing the issue now because it's served its purpose :).
